rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -17,6 +17,19 @@
 class PPOTorchRLModule(TorchRLModule, PPORLModule):
     framework: str = "torch"
 
+    @override(TorchRLModule)
+    def get_state(self, inference_only: bool = False) -> Mapping[str, Any]:
```

Comment:
So basically change the signature to Dict (also in the parent method). We don't have to do all Mapping -> Dict in this PR, but we should not introduce new `Mapping` occurrences anymore in new PRs.